src/workerd/api/streams/queue.h

```diff
@@ -599,7 +599,7 @@ class ValueQueue final {
     void reject(jsg::Lock& js, jsg::Value& value);
   };
 
-  class Entry final: public kj::Refcounted {
+  class Entry {
```

Comment:
What's the motivation for this commit? I would assume refcounting would be cheaper than allocating-on-clone.

Reply:
discussed in chat :-)